.vscode/settings.json

```diff
@@ -1,6 +1,25 @@
 {
+  "C_Cpp.autoAddFileAssociations": false,
   "files.associations": {
-    "*.overlay": "dts",
```

Comment:
The ZMK Tools extension now does this automatically, so a setting is not necessary.